sorting/java/SelectionSort.java

```diff
@@ -0,0 +1,50 @@
+class Selection

```

Comment:
Rename file's name from `SelectionSort` to `selection-sort`

Reply:
is it really necessary to do all these additional changes? i have done all the necessary changes that were required

Reply:
Yes, because we should write the code below the standard, and it will see like one people write all the code

Reply:
yeah i have renamed the java files. just left with the cpp and c programs

Reply:
Why did you only update the java file?